packages/components/src/LightBox/LightBox.css

```diff
@@ -18,3 +18,61 @@
 .wrapper:global(.ril__outer .ril__toolbarItem) {
```

Comment:
Do we still need this? I _assume_ this is coming from the old package

Reply:
Nice catch!